muswarmadmin/__main__.py

```diff
@@ -1,28 +1,39 @@
+import requests
+import logging
+import asyncio
+
```

Comment:
I usually separate with one space the imports from external modules with the import of internal modules.